tox.ini

```diff
@@ -29,6 +31,10 @@ deps =
     -r{toxinidir}/requirements-testing.txt
 commands = {env:_PYTEST_CMD:pytest} {env:_PYTEST_MORE_ARGS:} {posargs:-vvl}
 
+; Black doesn't support >py38 now
 [testenv:py38-pytestlatest-linters]
 deps = black
 commands = black --check --verbose setup.py docs pytest_bdd tests
+
+[testenv:py36-pytestlatest-coverage]
+setenv = PYTHONIOENCODING = utf-8
```

Comment:
what is this for?

Reply:
There are a few failing tests in the master branch only for py36-pytestlatest-coverage. I found that this environment variable solves the issue. There were a few Unicode fixes in the 3.7 python release, but I didn't investigate deep, because have more prior tasks.

Reply:
Or I could create an issue but seems that priority will be very low and finally it won't be fixed until 3.6 python would be dropped

Reply:
Removed; Workaroundnadded to failing test